pkgs/development/tools/build-managers/bazel/bazel_4/default.nix

```diff
@@ -667,7 +664,7 @@ stdenv.mkDerivation rec {
   # Save paths to hardcoded dependencies so Nix can detect them.
   postFixup = ''
     mkdir -p $out/nix-support
-    echo "${customBash} ${defaultShellPath}" >> $out/nix-support/depends
+    echo "${defaultShellPath}" >> $out/nix-support/depends
     # The templates get tar’d up into a .jar,
```

Comment:
Specifying `python27` and `python3` isn't necessary anymore since they're part of `defaultShellPath`.